eclair-core/src/test/scala/fr/acinq/eclair/io/PeerConnectionSpec.scala

```diff
@@ -151,7 +151,7 @@ class PeerConnectionSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike wi
     probe.watch(transport.ref)
     probe.send(peerConnection, PeerConnection.PendingAuth(connection.ref, Some(remoteNodeId), address, origin_opt = Some(origin.ref), transport_opt = Some(transport.ref)))
     transport.send(peerConnection, TransportHandler.HandshakeCompleted(remoteNodeId))
-    probe.send(peerConnection, PeerConnection.InitializeConnection(peer.ref))
+    probe.send(peerConnection, PeerConnection.InitializeConnection(peer.ref, nodeParams.chainHash, nodeParams.features, doSync = true))
```

Comment:
We're not testing `doSync = false` anywhere, maybe one of these tests should set it to `false`?

Reply:
Yes we are in the `connect` method, which has `doSync: Boolean = false`.

Reply:
Great, I missed that